modules/unsupported/process-geometry/src/main/java/org/geotools/process/geometry/PolyLabeller.java

```diff
@@ -54,9 +54,9 @@ static Geometry getPolylabel(Geometry polygon, double precision) {
             throw new IllegalStateException("Can not label empty geometries");
         }
 
-        if (!polygon.isValid()) {
+       /* if (!polygon.isValid()) {
```

Comment:
if we're sure that validity check is not required anymore why do we keep it commented?